drivers/wireless/ieee80211/bcm43xxx/bcmf_driver.c

```diff
@@ -1744,3 +1744,39 @@ int bcmf_wl_get_ssid(FAR struct bcmf_dev_s *priv, struct iwreq *iwr)
 
   return ret;
 }
+
+int bcmf_wl_set_country_code(FAR struct bcmf_dev_s *priv,
+                             int interface, FAR void *code)
+{
+  uint8_t country[4] =
+    {
+    };
+
+  uint32_t out_len;
+
+  memcpy(country, code, 2);
+
+  /* Why out_len = 4 ? Padding bytes to ensure array is
+   * terminating with null byte
+   */
+
+  out_len = sizeof(country);
+
+  return bcmf_cdc_iovar_request(priv, interface, true,
```

Comment:
Question: is the return of bcmf_cdc_iovar_request() compatible with POSIX error number? Otherwise I think it should be important to have some debug error/warning here to indicate the right failure, i.e.: unknown country code, etc...

Reply:
Thanks for the question, at present implement, the control commands of cdc all use POSIX compatible errono return values. The private control state only log printed:

```
cp> wapi country wlan0 TE
ioctl(0x8b37): 22
ERROR: Process command (country) failed.
cp> [01/01 00:01:08.610412] [25] [cp] bcmf_cdc_control_request_unsafe: Invalid cdc status 0xfffffffe
```
https://github.com/apache/incubator-nuttx/blob/master/drivers/wireless/ieee80211/bcm43xxx/bcmf_cdc.c#L230

We will consider better mapping of driver errors into POSIX compatible ones in the next patch, as you mentioned, which will better help the user to understand what exactly happened in the driver